File: Tarea7/src/Employee.java
import java.util.ArrayList;

public class Employee {

    private String name;
    private String lastName;
    private ArrayList<BankAccount> accountList;

    public Employee(String name, String lastName) {
        this.name = name;
        this.lastName = lastName;
        this.accountList = new ArrayList<>();
    }


    public String getName() {
        return name;
    }
    public String getLastName() {
        return lastName;
    }
    public String getFullName(){
        System.out.println(name+" "+lastName);
        return ("");
    }
    public ArrayList<BankAccount> getAccountList() {
        return accountList;
    }

    public void addAccount(BankAccount account) {
        accountList.add(account);
    }

    public String getDatosUsuario (){
        System.out.println("Titular de la cuenta: "+getName()+ " "+getLastName());
        System.out.println(accountList);
        return ("");
    }
}

File: Tarea7_python/Employee.py
class Employee:
  __name=""
  __last_name=""
  __account_list=[]
  
  def __init__(self, name, last_name):
      self.__name = name
      self.__lastName = last_name
      self.__account_list = []

  def get_name(self):
      return self.__name

  def get_last_name(self):
      return self.__last_name

  def get_full_name(self):
      print(self.__name + " " + self.__last_name)
      return ""

  def get_account_list(self):
      return self.__account_list

  def add_account(self, account):
      self.__account_list.append(account)

  def get_datos_usuario(self):
      print("Titular de la cuenta:", self.get_name(), self.get_last_name())
      for account in self.__account_list:
          print(account.get_datos())
      return ""